tutorials/scripting/gdscript/gdscript_styleguide.rst

```diff
@@ -620,7 +620,29 @@ Naming conventions
 
 These naming conventions follow the Godot Engine style. Breaking these will make
 your code clash with the built-in naming conventions, leading to inconsistent
-code.
+code. As a summary table:
+
++---------------+----------------+----------------------------------------------------+
+| Type          | Convention     | Example                                            |
++===============+================+====================================================+
+| File names    | snake_case     | ``yaml_parser.gd``                                 |
++---------------+----------------+----------------------------------------------------+
+| Class names   | PascalCase     | ``class_name YAMLParser``                          |
++---------------+----------------+----------------------------------------------------+
+| Node names    | PascalCase     | ``const Weapon = preload("res://weapon.gd")``      |
++---------------+----------------+----------------------------------------------------+
+| Functions     | snake_case     | ``func load_level():``                             |
++---------------+----------------+----------------------------------------------------+
+| Variables     | snake_case     | ``particle_effect``                                |
++---------------+----------------+----------------------------------------------------+
+| Signals       | snake_case     | ``signal door_opened`` (always in past tense)      |
```

Comment:
I feel "Always" hints that it is even more mandatory than usual, whereas even a few built-in signals don't follow this convention.
```suggestion
| Signals       | snake_case     | ``signal door_opened`` (in past tense)             |
```